doc-content/enterprise-only/release-notes/rn-known-issues-ref.adoc

```diff
@@ -89,169 +70,245 @@ Workaround: Use the legacy adapter instead of the Elytron adapter. Enter the fol
+Issue: If the properties panel for a process is open and then you maximize and restore the panel, it closes.
 
-Workaround: Ensure one copy has all of the assets, then remove all other copies. If {CENTRAL} performance is slow, restart all of the nodes connected to it.
+Steps to reproduce:
 
-//approved
-.In a high-availability authoring environment, project creation fails when a {CENTRAL} node is down [https://issues.jboss.org/browse/RHPAM-2475[RHPAM-2475]]
+. Create a process.
+. Open the properties panel for the process.
+. Click the image:ReleaseNotes/maximize-button.png[] *Maximize* button to maximize the panel to the size of the entire screen.
+. Click the image:ReleaseNotes/minimize-button.png[] *Maximize* button to restore the panel to its previous size.
 
-Issue: When you create a project, the project creation does not complete. The cause is one {CENTRAL} node in a high-availability authoring environment going down.
+Expected result: The properties panel remains open.
 
-Workaround: Create the project again.
+Actual result: The properties panel closes.
 
-//approved
-.In a high-availability authoring environment, asset creation is not completed when a {CENTRAL} node is down [https://issues.jboss.org/browse/RHPAM-2476[RHPAM-2476]]
+Workaround: Open the properties panel again.
 
-Issue: When you create an asset, the asset is not indexed. It is displayed in {CENTRAL} but you cannot use it. The cause is one {CENTRAL} node in a high-availability authoring environment going down.
+.If the properties panel is opened, after you switch to a different view, the panel closes [https://issues.redhat.com/browse/RHPAM-2612[RHPAM-2612]]
 
-Workaround: In the *Project Explorer* view, open the asset and click *Save*.
+Issue: If the properties panel for a process is open and you switch to a different view, the panel closes.
 
-//approved
-.In a high-availability authoring environment, multi-project import slows down {CENTRAL} [https://issues.jboss.org/browse/RHPAM-2477[RHPAM-2477]]
+Steps to reproduce:
 
-Issue: When a user imports multiple projects in a high-availability {CENTRAL} with multi-project import enabled, {CENTRAL} slows down and consumes significant resources.
+. Create a process.
+. Open the properties panel for the process.
+. Switch the view in one of the following ways:
+** Click the process name in the breadcrumb trail in the top part of the window
+** Click the process in a drop-down menu
+** Switch to the *Documentation* tab and then switch back to the *Model* tab.
+
+Expected result: The properties panel remains open.
+
+Actual result: The properties panel closes.
+
+Workaround: Open the properties panel again.
+
+.When you create a gateway, options for the *Default route* property might not be available [https://issues.redhat.com/browse/RHPAM-2536[RHPAM-2536]]
+
+Issue: In some cases, when you create a gateway, no options are available in the drop-down list for the *Default route* option.
```

Comment:
remove comma after gateway

This isn't clear. I don't find the drop-down list referenced. Kris can probably help you clarify. Doesn't make sense as is.

Reply:
OK, after hunting around a bit more I found it in the Properties panel. So I suggest:
 Sometimes when you create a gateway, no options are available from the *Default Route* drop-down list in the *Properties* panel.

Reply:
At this stage, I would make title and description the same. Note that Route is capped.

Reply:
I think "sometimes" sounds colloquial here, and what is wrong with "in some cases"?

Reply:
In some cases is fine.